lightning/tidb-lightning.toml

```diff
@@ -114,6 +114,19 @@ strategy = ""
 backend = "importer"
 # Address of tikv-importer when the backend is 'importer'
 addr = "127.0.0.1:8287"
+
+# The `duplicate-resolution` parameter is deprecated starting from v8.0.0 and will be removed in a future release. For more information, see <https://docs.pingcap.com/tidb/dev/tidb-lightning-physical-import-mode-usage#the-old-version-of-conflict-detection-deprecated-in-v800>.
+# Whether to detect and resolve duplicate records (unique key conflict) in the physical import mode.
+# The following resolution algorithms are supported:
+#  - none: does not detect duplicate records, which has the best performance of the two algorithms.
+#          But if there are duplicate records in the data source, it might lead to inconsistent data in the target TiDB.
+#  - remove: if there are primary key or unique key conflicts between the inserting data A and B,
+#            A and B will be removed from the target table and recorded
+#            in the `lightning_task_info.conflict_error_v1` table in the target TiDB.
+#            You can manually insert the correct records into the target table based on your business requirements.
+#            Note that the target TiKV must be v5.2.0 or later versions; otherwise it falls back to 'none'.
+# The default value is 'none'.
+# duplicate-resolution = 'none'
```

Comment:
recovering old config? there is a `record` option

Reply:
I deleted the old config in https://github.com/pingcap/tidb/pull/51710
But we should mark them as deprecated rather than directly delete them, so I add them back here marked as deprecated.
https://docs.pingcap.com/tidb/v8.0/tidb-lightning-configuration
The doc also marks them as deprecated rather than directly delete them.

The `record` option was only for internal use and was not declared in the older documentation for users, so I think we do not need to add it here either.
https://docs.pingcap.com/tidb/stable/tidb-lightning-configuration

Reply:
"deprecated" means the configuration value is not recommended but still takes effect. But `duplicate-resolution = "remove"` does not take effect.